### PR TITLE
Use docker-compose healthcheck

### PR DESCRIPTION
**Pull request by vsumner**
I updated the following:
* Healthcheck to wait for db to be ready
* networking the containers
* data volume for postgres to keep changes

Closes # for release #.

### CODE REVIEW

**Comment by EndermanAPM** [comment]
Does this work well on environments where the postgres is provided by the platform?
For example, when it gets deployed to heroku by travis.

**Comment by LeonardoGentile** [comment]
any progress on this?

File: {{cookiecutter.github_repository_name}}/docker-compose.yml [hunk 1]
```diff
@@ -27,3 +38,11 @@ services:
...
     ports:
       - "8001:8001"
+    networks:
+      - backend
+ 
+volumes:
+  db-data:
+
+networks:
+  backend:
```
Comment (agconti) [inline {{cookiecutter.github_repository_name}}/docker-compose.yml:48]: hey @vsumner thanks for adding this! ✨ 

Why did you add a network for backend and a specific volume for postgres?

Comment (vsumner) [reply {{cookiecutter.github_repository_name}}/docker-compose.yml:48]: Thats not required. I use it to keep my test data between down and ups. 
You can then destroy the data with [`docker-compose down -v`](https://docs.docker.com/compose/reference/down/).

Comment (vsumner) [reply {{cookiecutter.github_repository_name}}/docker-compose.yml:48]: The backend isn't required for that, the volume is.

I don't remember if the backend is needed.
